tests/providerTest.js

```diff
@@ -223,9 +204,13 @@ describe('provider testing', () => {
               ...applicationFormUrlEncoded
             }
           })
-        chai.assert.strictEqual(
-          result.status, 200, `creating user '${parameters.username}' failed`
-        )
+
+        const { message } = getOCSMeta(result)
+        if (result.status !== 200 && message !== 'account already exists') {
+          chai.assert.strictEqual(
+            result.status, 200, `creating user '${parameters.username}' failed`
+          )
+        }
```

Comment:
recreating a user should work but sometimes it fails with `account already exists` mostly with share user `testABC`.
So have to add this code block.

But will also work with this delete code if it is not to removed:
https://github.com/owncloud/owncloud-sdk/blob/8623274eed976753b03a6d31572d44b29b18b5a4/tests/providerTest.js#L213-L215

Reply:
Related issue in ocis: https://github.com/owncloud/ocis/issues/2027

Reply:
the user should have done at least a request to `/ocs/v2.php/cloud/capabilities` , see the hack below. Maybe that is not enough.
Can we change the hack below an make it a webdav `PROPFIND` request? will that create the user correctly or do we need to create a file / folder?

Reply:
@individual-it We need to make any requests twice and then only the user will be deleted successfully. (requesting `capabilities` twice will work)
Should we do that?

Reply:
yes, let's try that and please mention that also in owncloud/ocis#2027

Reply:
Comment added to the issue: https://github.com/owncloud/ocis/issues/2027#issuecomment-834116526